### ISSUE TITLE
Rendering Reports

### ISSUE THREAD

**Issue by amandawmosaic** [MosaicGroups/mosaic-groups#105]
Is it possible to have groups reports rendered in a spreadsheet format?

**Comment by jeffklassen** [comment]
Yep. This was not in your required list of changes to be made before we re-work the UI, so we will table this for the time being.

**Comment by amandawmosaic** [comment]
Sorry- forgot I asked
